apps/storefront/src/store/slices/company.ts

```diff
@@ -35,7 +35,7 @@ const initialState: CompanyState = {
     firstName: '',
     lastName: '',
     emailAddress: '',
-    customerGroupId: 1,
+    customerGroupId: 0,
```

Comment:
hey @BrianJiang2021 only one comments, could you please add a comment here so anyone reading this understands why this should be set up with 0 as default?